### ISSUE TITLE
Not able to browse drama list

### ISSUE THREAD

**Issue by long-lo** [groggyegg/plugin.video.dramacool#23]
Kodi Build: 19.3.0

When trying to view the list of Chinese dramas, I'm getting an error to check the logs. 

Upon checking the logs, this is the error returned:

```
2022-01-22 14:12:24.413 T:5116    ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'peewee.OperationalError'>
                                                   Error Contents: too many SQL variables
                                                   Traceback (most recent call last):
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\script.module.peewee\.\peewee.py", line 3160, in execute_sql
                                                       cursor.execute(sql, params or ())
                                                   sqlite3.OperationalError: too many SQL variables
                                                   
                                                   During handling of the above exception, another exception occurred:
                                                   
                                                   Traceback (most recent call last):
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\plugin.video.dramacool\resources\lib\addon.py", line 322, in <module>
                                                       plugin.run()
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\plugin.video.dramacool\resources\lib\plugin.py", line 51, in run
                                                       function(**kwargs)
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\plugin.video.dramacool\resources\lib\addon.py", line 172, in _
                                                       plugin.redirect(plugin.full_path + '&filterstr=' + quote(dumps(dialog.result())))
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\plugin.video.dramacool\resources\lib\plugin.py", line 10, in redirect
                                                       run()
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\plugin.video.dramacool\resources\lib\plugin.py", line 51, in run
                                                       function(**kwargs)
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\plugin.video.dramacool\resources\lib\addon.py", line 189, in _
                                                       for drama in Drama.select().where(Drama.path << paths):
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\script.module.peewee\.\peewee.py", line 6923, in __iter__
                                                       self.execute()
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\script.module.peewee\.\peewee.py", line 1911, in inner
                                                       return method(self, database, *args, **kwargs)
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\script.module.peewee\.\peewee.py", line 1982, in execute
                                                       return self._execute(database)
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\script.module.peewee\.\peewee.py", line 2155, in _execute
                                                       cursor = database.execute(self)
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\script.module.peewee\.\peewee.py", line 3173, in execute
                                                       return self.execute_sql(sql, params, commit=commit)
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\script.module.peewee\.\peewee.py", line 3167, in execute_sql
                                                       self.commit()
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\script.module.peewee\.\peewee.py", line 2933, in __exit__
                                                       reraise(new_type, new_type(exc_value, *exc_args), traceback)
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\script.module.peewee\.\peewee.py", line 191, in reraise
                                                       raise value.with_traceback(tb)
                                                     File "C:\Users\longl\AppData\Roaming\Kodi\addons\script.module.peewee\.\peewee.py", line 3160, in execute_sql
                                                       cursor.execute(sql, params or ())
                                                   peewee.OperationalError: too many SQL variables
                                                   -->End of Python script error report<--
                                                   
2022-01-22 14:12:24.488 T:5116     INFO <general>: Python interpreter stopped
2022-01-22 14:12:24.506 T:6016    ERROR <general>: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.dramacool/category/chinese-drama?label=33204
2022-01-22 14:12:24.522 T:15048   ERROR <general>: CGUIMediaWindow::GetDirectory(plugin://plugin.video.dramacool/category/chinese-drama?label=33204) failed
```

Thanks.

**Comment by groggyegg** [comment]
The issue has been fixed in dev release, v2.3.0. Will be merged to stable release after testing.

**Comment by groggyegg** [comment]
Fixed in v2.3.x
